great_expectations/init_notebooks/spark/validation_playground.ipynb

```diff
@@ -154,10 +154,81 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
    "outputs": [],
    "source": [
     "#validation_result"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## 5. Validation Operators\n",
+    "\n",
+    "The `validate` method evaluates one batch of data against one expectation suite and returns a dictionary of validation results. This is sufficient when you explore your data and get to know Great Expectations.\n",
+    "When deploying Great Expectations in a **real data pipeline, you will typically discover additional needs**:\n",
+    "\n",
+    "* validating a group of batches that are logically related\n",
+    "* validating a batch against several expectation suites such as using a tiered pattern like `warning` and `failure`\n",
+    "* doing something with the validation results (e.g., saving them for a later review, sending notifications in case of failures, etc.).\n",
+    "\n",
+    "`Validation Operators` provide a convenient abstraction for both bundling the validation of multiple expectation suites and the actions that should be taken after the validation.\n",
+    "\n",
+    "[Read more about Validation Operators in the tutorial](https://docs.greatexpectations.io/en/latest/tutorials/validate_data.html?utm_source=notebook&utm_medium=validate_data#save-validation-results)"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# This is an example of invoking a validation operator that is configured by default in the great_expectations.yml file\n",
+    "\n",
+    "#Generate a run id, a timestamp, or a meaningful string that will help you refer to validation results. We recommend they be chronologically sortable.\n",
+    "# Let's make a simple sortable timestamp. Note this could come from your pipeline runner (e.g., Airflow run id).\n",
+    "run_id = datetime.utcnow().isoformat().replace(\":\", \"\") + \"Z\"\n",
+    "\n",
+    "results = context.run_validation_operator(\n",
+    "    \"action_list_operator\", \n",
+    "    assets_to_validate=[batch], \n",
+    "    run_id=run_id)\n",
+    "\n"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## 6. View the Validation Results in Data Docs\n",
+    "\n",
```

Comment:
Quick confirmation: is this intended? If so, minor note that the run_id format should be updated.

Reply:
blaaaarg. Unintentional. FIxed.